native/build.gradle

```diff
@@ -65,12 +65,15 @@ if ((rootProject.name.contains('native') && !rootProject.hasProperty('skipNative
       flavour = out.toString().replaceAll('[\r\n\f]', '').toLowerCase()
     }
     if (flavour.startsWith('ubuntu14') || flavour.startsWith('ubuntu16') ||
+        flavour.startsWith('elementary') ||
+        flavour.startsWith('neon') ||
```

Comment:
Don't you need version checks here because not all versions of elementary OS or neon will work. Will support 14.04 or 16.04 compatible versions only. Likewise for fedora it will be RHEL6/7 compatible versions only.

At least the check should be for versions having >= 14.04 compatibility for ubuntu or >= RHEL6 compatibility for redhat.